test/tsconfig.json

```diff
@@ -2,7 +2,10 @@
 	"extends": "../tsconfig.json",
 	"compilerOptions": {
 		"paths": {
-			"@banners/*": [ "banners/*" ]
-		}
+			"@banners/*": [ "banners/*" ],
+			"@test/*": [ "test/*" ],
+			"@src/*": [ "src/*" ],
+			"@environment/*": [ "src/environment/prod/*", "src/environment/dev/*" ]
+		},
```

Comment:
Please remove the trailing comma here, otherwise strict JSON parsing might complain